preset/deep-learning/Dockerfile

```diff
@@ -261,7 +262,7 @@ RUN conda create -yn 'tensorflow-cpu' \
         "python=${PYTHON_VERSION}" \
         "dpnp=${DPNP_VERSION}" \
         "intel-extension-for-tensorflow=${ITEX_CPU_VERSION}=*cpu*" \
-        "intel-optimization-for-horovod=${HOROVOD_VERSION}" \
+        # "intel-optimization-for-horovod=${HOROVOD_VERSION}" \
```

Comment:
Can you also comment out the label in your compose file so the docs reflect that change?